network/onchain-discovery/src/client.rs

```diff
@@ -0,0 +1,468 @@
+// Copyright (c) The Libra Core Contributors
```

Comment:
in your commit, call this out as a simple move refactor to save time for the reviewer.